.github/workflows/publish.yml

```
@@ -26,39 +26,55 @@ jobs:
         run: echo "CONTAINER_ID=$(docker create ${{ inputs.container-image }})" >> "$GITHUB_OUTPUT"
       - name: Move container files into GHA Workspace
         run: docker cp ${{ steps.create-container.outputs.CONTAINER_ID }}:/opt/app/dotcom-rendering/dotcom-rendering/. ${{ github.workspace }}
-      # We want to avoid uploading the uncompressed files to our riffraff bucket
+
+        # We want to avoid uploading the uncompressed files to our riffraff bucket
       # so we create our tarball outside the rendering/ folder, then delete the contents of rendering/
       # and finally move the tarball into rendering
-      - name: Compress DCR server files
+      - name: Compress existing DCR server files
         run: |
           tar -zcf rendering.tar.gz rendering
           rm -r rendering/*
           mv rendering.tar.gz rendering/
-      - name: Compress DCR front-web server files
+
+      - name: Compress new DCR server files
```

would this be accurate? to avoid 'new' and 'old' naming. But if it's misleading don't worry about it
```suggestion
      - name: Compress article DCR server files
```